src/Uno.UI/DirectUI/DateComparer.cs

```diff
@@ -103,10 +103,7 @@ public void SetCalendarForComparison(Calendar pCalendar)
 
 			global::System.Diagnostics.Debug.Assert(m_spCalendar is { });
 
-			lhs = lhs.ToUniversalTime(); // UNO
-			rhs = rhs.ToUniversalTime(); // UNO
-
-			long delta = lhs.Ticks - rhs.Ticks;
+			long delta = lhs.ToUniversalTime().Ticks - rhs.ToUniversalTime().Ticks;
```

Comment:
Even if I agree that this syntax is better, as this file is a port of the WinUI source code, I would keep the previous version to make clearer what has modified for uno.

Reply:
@dr1rrb This change brings back the original WinUI source, and is a behavioral change.

Reply:
Oh indeed, the `lhs` and `rhs` are used in code below, I thought they were used only for computation of the `delta` ...